templates/hooks/table/sdk_create_post_set_output.go.tpl

```diff
@@ -2,4 +2,8 @@
 		if err := rm.syncTTL(ctx, desired, &resource{ko}); err != nil {
 			return nil, err
 		}
+	}
+
+	if desired.ko.Spec.ContributorInsights != nil {
+		ackcondition.SetSynced(&resource{ko}, corev1.ConditionFalse, nil, nil)
 	}
```

Comment:
table creation is usually fast (5-10seconds), do we wanna do a short requeue instead?